test/madetomeasure_api/test/routes/subscribers.clj

```diff
@@ -11,6 +11,14 @@
     (body body-string)
     (content-type "application/json")))
 
+(defn json-delete [endpoint]
+  (->
+    (request :delete endpoint)
+    (content-type "application/json")))
+
+(def subscriber
```

Comment:
Any thoughts on how to use fixtures with clojure?

Not entirely sure how to do this. But really I'd love to have this wrapped up in a transaction.